integrationtest/helpers.go

```diff
@@ -410,27 +409,36 @@ func makeRules(t *testing.T, rules string) []rbacv1.PolicyRule {
 	return policyRules.Rules
 }
 
-func makeExpectedLabels(t *testing.T, userMetadata map[string]interface{}) map[string]string {
+func makeExpectedLabels(t *testing.T, extraLabels map[string]interface{}) map[string]string {
 	t.Helper()
 
-	userLabels := map[string]string{}
-	err := mapstructure.Decode(userMetadata["labels"], &userLabels)
-	require.NoError(t, err)
-
 	expectedLabels := map[string]string{}
-	if userLabels != nil {
-		expectedLabels = combineMaps(userLabels, standardLabels)
+	if extraLabels != nil {
+		expectedLabels = combineMaps(asMapString(extraLabels), standardLabels)
 	} else {
 		expectedLabels = standardLabels
 	}
 	return expectedLabels
 }
 
-func makeExpectedAnnotations(t *testing.T, userMetadata map[string]interface{}) map[string]string {
-	t.Helper()
+func makeExpectedAnnotations(extraAnnotations map[string]interface{}) map[string]string {
```

Comment:
This function doesn't seem necessary anymore?

Reply:
😄 good point